### PR TITLE
New: Drag image from desktop [ED-3817]

### PR DESCRIPTION
**Pull request by YitzhakAvm**

### CODE REVIEW

**Review by matipojo — CHANGES_REQUESTED** [review]
Please fix the conflict

File: includes/widgets/video.php [hunk 1]
```diff
@@ -1174,7 +1174,7 @@ private function render_hosted_video() {
 
 		$video_params = $this->get_hosted_params();
 		?>
-		<video class="elementor-video" src="<?php echo esc_url( $video_url ); ?>" <?php Utils::print_html_attributes( $video_params ); ?>></video>
+		<video class="elementor-video" src="<?php echo esc_attr( $video_url ); ?>" <?php Utils::print_html_attributes( $video_params ); ?>></video>
```
Comment (matipojo) [inline includes/widgets/video.php:1177]: Why?

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/widgets/video.php (hunk 1 above)
Comment (ItzhakAvm) [reply includes/widgets/video.php:1177]: Because it's not always a URL, when in preview mode - it's a base64 string.

Comment (matipojo) [reply includes/widgets/video.php:1177]: Please add a comment in the code

**Pull request merged**